propagation.go

```diff
@@ -169,6 +164,107 @@ func (p *textMapPropagator) Extract(abstractCarrier interface{}) (SpanContext, e
 	return ctx, nil
 }
 
+type zipkinB3Propagator struct {
```

Comment:
I would like zipkin to be in its own `./zipkin/` package. There will be more stuff there in the future once we upgrade the client to the new non-zipkin thrift model. the injector/extractor interfaces are public, so this propagator does not need to be in this package.